hw/ip/adc_ctrl/dv/cov/adc_ctrl_cov_unr_excl.el

```diff
@@ -0,0 +1,99 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Manually selected condition coverage UNR items from auto-generated UNR file to help reach V2
+// coverage metrics.
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: chencindy
+// Format Version: 2
+// Date: Thu Apr 21 15:18:07 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2208754432 2034735209"
+INSTANCE: tb.dut.u_reg.u_adc_en_ctl_cdc
+ANNOTATION: "[UNR]: busy is always 1 when src_req is high."
```

Comment:
Hi @msfschaffner,
This UNR and below are related to CDC code: https://github.com/lowRISC/opentitan/blob/master/hw/ip/prim/rtl/prim_reg_cdc.sv#L85
It says when `src_req`= 1 `busy`=  0 is unreachable.

Reply:
@msfschaffner does this argue for changes in the logic?

Reply:
@cindychip can you double check whether the tool actually refers to the entire `!busy` expression as opposed to the `busy` signal alone?

Otherwise this does not make sense... i.e., the design naturally places requests (`src_req == 1`) when the CDC synchronizer is NOT busy (`busy == 0`), and if it didn't that would be a bug.

On the other hand, the register top that instantiates `prim_reg_cdc` ensures that it will never write to a busy synchronizer, hence `src_req == 1` and `!busy == 0` would make sense...

Reply:
Let me double check! Thanks Michael!

Reply:
Sorry! Double checked the description below:
`Condition 2 "2823382532" "(src_req && ((!busy))) 1 -1" (2 "10")`
The value it refers to is `10`.
Yes @msfschaffner you are right, sorry for the confusion - it is `src_req = 1` and `!busy` = 0.

Reply:
Ok in that case this makes sense and these exclusions are all ok.

As for @matutem's point: 
I don't think we can do much about this... the instantiating design makes sure that it never writes to a busy synchronizer, and I don't think we want to loosen that logic...

Reply:
Thanks again @msfschaffner and @matutem. I am fine with not loosen the logic. Might just need to do more exclusion for CDC modules.
Also attached the screenshot of the UNR GUI. Sorry again for the missing `!`.
https://screenshot.googleplex.com/M76joErC9UjYYyC

Reply:
No worries!